docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,325 @@
+
+  </template>
+  <template slot="hint">
+
+  This is the question hint, you can use **markdown** here as well.
+
+  Paragraph **2**
+
+  </template>
 </question>
 </variable>
 </include>
 
-**Use the `has-input` attribute to add a text input box for users to enter their answer.**
+****Common `<question>` Options and Slots****
+Name | Type | Default | Description
+--- | --- | --- | ---
+header | `String` | `''` | The markup to insert into the card header. The card header is omitted if this is not provided.
+header | Slot | empty | Functions similar to the `header` option, and overrides it if both are present.
+hint  | `String` | `''` | The content to display in the hint box.
+hint  | Slot | empty | Functions similar to the `hint` option, and overrides it if both are present.
+
+
+#### Mcq and Checkbox Questions {.mt-2}
+
+Mcq and checkbox questions are indicated with the `type="mcq"` or `type="checkbox"` attribute.
+
+In both instances, you can include the possible answers using the `<q-option>` component. Simply place it inside the <tooltip content="if you wish, you could place it in the card header slot mentioned above as well!">question</tooltip>. To indicate the correct answer(s), add the `correct` attribute to the `<q-option>` component(s).
+
+Optionally, you can provide the reason for the particular option using the `<q-option reason="...">` attribute, or the `<template slot="reason">` slot (similar to the `hint` / `header` options).
+
+**Mcq Question**
 
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">html</variable>
 <variable name="code">
-<question has-input>
-  What is the distance from Earth to the Moon in kilometers? (Give your answer in 3 s.f)
-  <div slot="hint">It is between 300,000 and 400,000 km</div>
-  <div slot="answer">384,000 km <md>:full_moon:</md></div>
+<question type="mcq" header="Which of these **contradicts** the heuristics recommended when creating test cases with multiple inputs?">
+<q-option reason="This is **correct**. We need to figure out if a positive test case works!">
+  Each valid test input should appear at least once in a test case that doesn’t have any invalid inputs.
+</q-option>
+<q-option>
+  It is ok to combine valid values for different inputs.
+</q-option>
+<q-option>
+  No more than one invalid test input should be in a given test case.
+</q-option>
+<!-- Use the 'correct' attribute to indicate an option as correct. -->
+<q-option correct>
+  All invalid test inputs must be tested together.
+  <!-- Optionally, you may use a reason slot instead of a reason attribute. -->
+  <template slot="reason">
+  If you test all invalid test inputs together, you will not know if each one of the invalid inputs are handled
+  correctly by the SUT.
+  This is because most SUTs return an error message upon encountering the first invalid input.
+  </template>
+</q-option>
+<template slot="hint">
+How do you figure out which inputs are wrong? (or correct)
+</template>
```

Comment:
![image](https://user-images.githubusercontent.com/1673303/88081441-f4ce4f00-cbb2-11ea-93b2-2b21a73de353.png)
Will this confuse the reader? This reminded me, do we want to support having multiple correct answers to an MCQ question.

Reply:
it's already supported, but undocumented.

Fixed the ticks too